### DISCUSSION TITLE
FLOSS Weekly?

### DISCUSSION THREAD

**Discussion by jp-bennett** [uutils/coreutils#6448] [General]
Hey, I'm the host of FLOSS Weekly, one of the oldest podcasts about Open Source. I'm a fan of the project, and wonder if one or two of the project leads would like to come talk about it on the podcast. Respond here or email at floss <at> hackaday <dot> com Let me know! 

Jonathan Bennett

**Comment by sylvestre** [comment]
With pleasure I will send you an email

**Comment by jp-bennett** [comment]
Really had a blast with this! Show is available at https://hackaday.com/2024/07/17/floss-weekly-episode-792-rust-coreutils/
